src/edfi-performance-test/edfi_performance_test/performance_tester.py

```diff
@@ -44,7 +45,7 @@ def monitor_fail_ratio(runner: Runner):
             )
 
 
-def spawn_pref_tests(args: MainArguments, user_class: Type[User]) -> None:
+def spawn_pref_tests(args: MainArguments, user_class: Type[HttpUser]) -> None:
```

Comment:
Can you make the same change here:

```
edfi_performance_test/performance_tester.py:106: error: Argument 2 to "spawn_pref_tests" has incompatible type "Type[VolumeTestUser]"; expected "Type[HttpUser]"
```

Reply:
Rather than using HttpUser, changed to FastHttpUser, which is the class VolumeTestUser extends